app/views/documents/_inline_attachment.html.erb

```diff
@@ -1,4 +1 @@
-<span id="attachment_<%= attachment.id %>" class="attachment-inline">
-  <%= link_to attachment.title, attachment.url %>
-  (<%= attachment_attributes(attachment) %>)
-</span>
+<span id="attachment_<%= attachment.id %>" class="attachment-inline"><%= link_to attachment.title, attachment.url %> (<%= attachment_attributes(attachment) %>)</span>
```

Comment:
In some ways this is a bit of a shame to put this on one line.
I find it less readable than what was there previously.

But i guess you need it this way as spans preserve line breaks?
🤔 and a helper seems OTT...

Reply:
Yeah, I tried just using `chomp` and putting it back over multiple lines, but both contribute to the additional space, and it's such a small file that anything else seemed like overkill.